apps/omg_watcher/test/support/test_helper.ex

```diff
@@ -46,4 +46,13 @@ defmodule OMG.Watcher.TestHelper do
   end
 
   def create_topic(main_topic, subtopic), do: main_topic <> ":" <> subtopic
+
+  def to_response_address(address) do
+    "0X" <> encoded =
```

Comment:
the capital `X` looks strange here

Reply:
True. I've seen a task to make all hex-ed values lowercase which should help here.